Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -481,7 +475,7 @@ - (NSString *)textInputShouldChangeText:(NSString *)text inRange:(NSRange)range
     }
   }
 
-  NSString *previousText = [backedTextInputView.attributedText.string copy] ?: @""; // TODO(OSS Candidate ISS#2710739)
```

Comment:
Was this macOS change pushed upstream and that's why we're removing our TODO?

Reply:
Yes, with 15dda0ab5a491dcc83539f9ef32c9896be41074a.